erpnext/selling/doctype/sales_order/sales_order.py

```diff
@@ -387,7 +387,7 @@ def get_work_order_items(self, for_raw_material_request=0):
 				else:
 					pending_qty = stock_qty
 
-				if pending_qty:
+				if pending_qty and not frappe.get_value("Product Bundle", {"new_item_code": i.item_code}):
```

Comment:
get_value in for loop will slow down the execution, better prepare the tuple of product bundle items based on self.items and then one by one check the item present in that tuple or not